internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -257,6 +265,32 @@ func (opts *Opts) providerAndRegionToConstant() {
 	opts.Region = strings.ReplaceAll(strings.ToUpper(opts.Region), "-", "_")
 }
 
+func (opts *Opts) defaultValues() error {
+	if !opts.defaultValue {
+		return nil
+	}
+
+	opts.SkipSampleData = true
+	opts.SkipMongosh = true
```

Comment:
is this on the spec? we should respect the flags default

Reply:
yes, we skip both with the default flag

Reply:
if this is in the spec I'm ok, my expectation would have been to enter mongosh in the end

Reply:
I am happy to change this behaviour and update the spec if you both agree

Reply:
I'm ok as it if it's on the spec, we can revisit later